perception/autoware_map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1462,7 +1461,10 @@ PredictedObject MapBasedPredictionNode::getPredictedObjectAsCrosswalkUser(
 
   // try to find the edge points for other surrounding crosswalks and generate path to the crosswalk
   // edge
-  for (const auto & crosswalk : external_surrounding_crosswalks) {
+  for (const auto & crosswalk : surrounding_crosswalks) {
+    if (crossing_crosswalk && crossing_crosswalk.get() == crosswalk) {
```

Comment:
Does this mean that `crossing_crosswalk` is necessary for computation cost?

Reply:
sorry for my missunderstanding

Reply:
crosswalk is optional, so I think it is necessary.